pkg/operation/botanist/component/logging/eventlogger/logging.go

```diff
@@ -25,6 +25,7 @@ const (
     Match               kubernetes.*` + v1beta1constants.DeploymentNameEventLogger + `*` + name + `*
     Operation           lift
     Nested_under        log
+    log_level           error
```

Comment:
Will it be like below, or it work in any format?
```suggestion
    Log_Level           error
```